br.com.wiselabs.simplexity.yml

```diff
@@ -0,0 +1,56 @@
+app-id: br.com.wiselabs.simplexity
```

Comment:
wiselabs.br.com redirects to br.com which seems the domain registrar? Do you control this?

If so how do I verify? Please put some webpage up or a txt file that links the submission.

Reply:
I still don't see the github linked anywhere on https://www.wiselabs.com.br/ is that available somewhere?

Reply:
> wiselabs.br.com redirects to br.com which seems the domain registrar? Do you control this?
> 
> If so how do I verify? Please put some webpage up or a txt file that links the submission.

Wiselabs Software is my software house company. This project is not part of core-business, but the site will be updated later to point Simplexity at PORTFOLIO page.